build_tools/docker/manage_images.py

```diff
@@ -49,6 +49,7 @@
     "emscripten": ["base"],
     "nvidia": ["base"],
     "riscv": ["base"],
+    "riscv-toolchain-builder": [],
```

Comment:
We don't necessarily have to host the docker in gcr.io, and I am happy to leave the manage_images/prod_digest change out of the PR. Please advise.

Reply:
Seems fine to host in the same place and include in the digest here.

We've had some ideas about rewording how Dockerfiles / images are maintained, but I don't know the status / timeline there.
* Define images in a shared location, e.g. https://github.com/nod-ai/base-docker-images/
* Build and push images using GitHub actions (using repository secrets), so any approved changes from community members can modify the files and not just any developer with access to the cloud project